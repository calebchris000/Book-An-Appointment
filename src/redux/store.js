import { configureStore } from "@reduxjs/toolkit";
import productDetailsReducer from "./productDetails/productDetailsSlice";
import productDescriptionReducer from "./productDescription/productDescriptionSlice";
import thunk from "redux-thunk";
import deleteCarReducer from './deleteCarSlice';

const store = configureStore({
  reducer: {
    productDetails: productDetailsReducer,
    productDescription: productDescriptionReducer,
  },
  middleware: [thunk],

  reducer: {
    deleteCar: deleteCarReducer,
  },
});

export default store;
