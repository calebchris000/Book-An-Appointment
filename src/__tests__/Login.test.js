import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../redux/store';
import LoginForm from '../components/Login/Login';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Router>
          <LoginForm loggedIn={false} />
        </Router>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});