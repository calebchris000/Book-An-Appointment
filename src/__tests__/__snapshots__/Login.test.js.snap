// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="login-container"
>
  <form
    className="login-form"
    onSubmit={[Function]}
  >
    <h1>
      Welcome Back!
    </h1>
    <label
      htmlFor="email"
    >
      Email:
    </label>
    <input
      onChange={[Function]}
      placeholder="Enter your email"
      required={true}
      type="email"
      value=""
    />
    <label
      htmlFor="password"
    >
      Password:
    </label>
    <input
      onChange={[Function]}
      placeholder="Enter your password"
      required={true}
      type="password"
      value=""
    />
    <button
      type="submit"
    >
      Log In
    </button>
  </form>
  <a
    href="/signup"
    onClick={[Function]}
  >
    Signup
  </a>
</div>
`;
